lib/collaborative_filtering.py

```diff
@@ -285,16 +284,13 @@ def train(self, item_vecs=None):
     def partial_train(self):
         """
         Train model for n_iter iterations. Can be called multiple times for further training.
-
         """
-        ctr = 1
-        while ctr <= self.n_iter:
+        for ctr in range(1, self.n_iter + 1):
```

Comment:
This is not really pull relevant to the pull request, let's make the pull requests modular. no need to change it back for now